bsp/src/mill/bsp/BSP.scala

```diff
@@ -64,11 +64,7 @@ object BSP extends ExternalModule {
     write(
       BspConfigJson(
         "mill-bsp",
-        Seq(
-          "sh",
-          "-c",
-          s"env ${sys.env.map { case (k, v) => s""""$k=$v"""" }.toSeq.mkString(" ")} $millPath -i ${BSP.getClass.getCanonicalName.split("\\$").last}/start"
-        ),
+        Seq(s"$millPath -i ${BSP.getClass.getCanonicalName.split("\\$").last}/start"),
```

Comment:
```suggestion
        Seq(s"$millPath -i ${BSP.getClass.getCanonicalName.split("\\$").head}/start"),
```

Although, this is effectively an array of length 1 and both will give the same result, semantically, we want the first part of the split, not the last.

Reply:
Good point